config/application.rb

```diff
@@ -32,7 +29,10 @@ def self.localized_email_sender
 
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults 7.0
```

Comment:
Note that jumping straight to the 7.0 defaults will make it impossible to revert back to 6.1, due to things like the [New ActiveSupport::Cache serialization format](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#new-activesupport-cache-serialization-format)

Reply:
Ok, thanks for the information! We will consider this when we do the merge!